src/pages/Home/Home.jsx

```diff
@@ -1,35 +1,65 @@
-import React from 'react'
-import axios from 'axios'
-
-import Photo from '../../components/Photo/Photo'
+import React from "react";
+import axios from "axios";
+import InfiniteScroll from "react-infinite-scroll-component";
+import Photo from "../../components/Photo/Photo";
 
 class Home extends React.Component {
-    state = {
-        photoData: null,
-    }
+   state = {
+      photoData: null,
+      page: 2,
+   };
+
+   loadInitialPhotos = async () => {
+      try {
+         const res = await axios(
+            `https://api.unsplash.com/photos?order_by=latest&page=1&client_id=${process.env.REACT_APP_API_KEY}`
+         );
+         const { data } = res;
+         this.setState({ photoData: data });
+      } catch (err) {
+         console.log(err);
+      }
+   };
 
-    loadRandomPhotos = async () => {
-        try {
-            const res = await axios(`https://api.unsplash.com/photos?order_by=latest&per_page=20&client_id=${process.env.REACT_APP_API_KEY}`)
-            const { data } = res
-            this.setState({ photoData: data })
-        } catch (err) {
-            console.log(err)
-        }
-    }
+   loadMorePhotos = async () => {
+      try {
+         const res = await axios(
+            `https://api.unsplash.com/photos?order_by=latest&page=${this.state.page}&client_id=${process.env.REACT_APP_API_KEY}`
+         );
+         const { data } = res;
+         this.setState({
+            photoData: this.state.photoData.concat(data),
+            page: this.state.page + 1,
```

Comment:
another way would be to increase the page with the scroll component, and in component did update you call the api when page number changes